### ISSUE TITLE
tacotron 학습 step 문의 드립니다

### ISSUE THREAD

**Issue by ocean-park** [hccho2/Tacotron-Wavenet-Vocoder#10]
tacotron 을 single 로 학습 중인데요, 20만 스텝을 넘어가고 있네요
tacotron의 학습 종료는 언제쯤일까요?
그리고 학습데이터는 (구글 음성인식으로 ) 10시간이 좀 안되구요
그런데 loss 가 아래와 같은데, 혹시 경험치랑 비슷한지 문의 드립니다.

Step 204027 [0.981 sec/step, loss=0.68417, avg_loss=0.66632]
Step 204028 [0.973 sec/step, loss=0.68508, avg_loss=0.66625]
Step 204029 [0.971 sec/step, loss=0.67858, avg_loss=0.66610]
Step 204030 [0.972 sec/step, loss=0.67207, avg_loss=0.66616]

**Comment by ocean-park** [comment]
참고~
wav 파일을 44k로 변경하고 새로 학습을 시작 했습니다
현재 타코는 아래와 같은 학습률을 보이고 있습니다
Step 262334 [1.364 sec/step, loss=0.64717, avg_loss=0.63480]

병렬로 학습하고 있는 wavenet 은
step 136361 - loss = 5.701, (1.038 sec/step)
이렇습니다.

**Comment by hccho2** [comment]
loss는 제가 했던 것과 비슷한 것 같습니다.
종료 여부는 중간에 생성되는 test 결과를 보고, 판단했습니다.


[2018-12-29 17:47:04.362]    Step 106259  [0.610 sec/step, loss=0.66996, avg_loss=0.64963]
[2018-12-29 17:47:05.190]    Step 106260  [0.600 sec/step, loss=0.72168, avg_loss=0.65171]
[2018-12-29 17:47:05.705]    Step 106261  [0.601 sec/step, loss=0.64711, avg_loss=0.65192]
[2018-12-29 17:47:06.080]    Step 106262  [0.595 sec/step, loss=0.61615, avg_loss=0.65112]
[2018-12-29 17:47:06.517]    Step 106263  [0.591 sec/step, loss=0.61003, avg_loss=0.65013]

**Comment by ocean-park** [comment]
내 답변 감사합니다 원본 데이터가 좋지 않은지 타코 트론 학습이 결과가 맘에 들지는 않네요 그래서 데이터를 다시 정제 해보고 새로 학습을 시작해보려고 합니다 답변 다시 한번 감사드립니다

**Comment by CHJ920319** [comment]
종료 여부는 중간에 생성되는 test 결과를 보고, 판단 하셨다고 했는데,
결과가 만족스러우면  그냥 강제로 종료를 시키는건가요?
스텝을 정해놓고 일정  스텝이 되면 종료가 되는 옵션이 없을까요?
num_steps 가 그런 옵션일줄 알았는데 설정 스텝보다 더 넘어가서 문의 드립니다!
